proto/aggkit/prover/v1/aggchain_proof_generation.proto

```diff
@@ -14,6 +14,12 @@ message GenerateAggchainProofRequest {
   uint64 start_block = 1;
   // The max end block for which the aggchain proof is requested.
   uint64 max_end_block = 2;
+  // L1 Info tree root.
+  bytes l1_info_tree_root = 3;
+  // L1 Info tree leaf.
+  bytes l1_info_tree_leaf = 4;
```

Comment:
What's the type of this one, is it a hash?

Reply:
yes, its hash

Reply:
Then can we have this instead ?

```suggestion
  bytes l1_info_tree_leaf_hash = 4;
```

What's about the `l1_info_tree_leaf_index` ?